tests/integration-tests/tests/update/test_update/test_update_awsbatch/pcluster.config.update.yaml

```diff
@@ -24,4 +24,4 @@ Scheduling:
           MinvCpus: 2
           MaxvCpus: 4
           DesiredvCpus: 3
-          SpotBidPercentage: 70
\ No newline at end of file
+          SpotBidPercentage: 0.7
```

Comment:
BidPercentage in Batch is an integer. why changing the type?

Reply:
Good catch! Fixed. We were using float for both SpotPrice and SpotBidPercentage